Type Error `TS2786` for `import('@wuespace/telestion-client-common').LoginDescription`

Currently, the `<LoginDescription />` component produces a type error when added as a child of `import('@wuespace/telestion-client-common').LoginPage`:

```
$ tsc --noEmit
src/components/login-page.tsx:14:5 - error TS2786: 'LoginDescription' cannot be used as a JSX component.
  Its return type 'Element | ReactElement<any, string | ((props: any) => ReactElement<any, any> | null) | (new (props: any) => Component<any, any, any>
)>[]' is not a valid JSX element.
    Type 'ReactElement<any, string | ((props: any) => ReactElement<any, any> | null) | (new (props: any) => Component<any, any, any>)>[]' is missing th
e following properties from type 'Element': type, props, key

14              <LoginDescription />
                 ~~~~~~~~~~~~~~~~


Found 1 error.
```

The project still runs using `tc-cli start`, but, of course, the error is, still, annoying.

This issue also occurs in the RocketSound PSC: https://github.com/TelestionTeam/telestion-daedalus2-psc/tree/e36c0a614132235945141ba4cc147f0e74628356

--- a/packages/telestion-client-common/src/components/pages/login-page/login-description.tsx
+++ b/packages/telestion-client-common/src/components/pages/login-page/login-description.tsx
@@ -64,7 +64,7 @@ export function LoginDescription({
 	description,
 	children
 }: LoginDescriptionProps) {
-	if (children) return children;
+	if (children) return <>{children}</>;
 
 	return (
 		<Text>

